src/Orc.NuGetExplorer.Xaml/Orc.NuGetExplorer.Xaml.Shared/Services/PackageCommandService.cs

```diff
@@ -137,14 +137,13 @@ private bool CanInstall(IPackageDetails package)
                 ValidatePackage(package);
             }
 
-            return package.IsInstalled != null && !package.IsInstalled.Value && package.GetErrorCount(ValidationTags.Api) == 0;
+            return package.IsInstalled != null && !package.IsInstalled.Value && package.ValidationContext?.GetErrorCount(ValidationTags.Api) == 0;
```

Comment:
ValidationContext should never be allowed null. Just make the setter private, ResetValidationContext will create a new instance.